consensus/service/config/src/tokens.rs

```diff
@@ -76,6 +76,35 @@ mod pem_signer_set {
     }
 }
 
+mod hex_signature {
```

Comment:
Here we'll actually need to be opinionated because we are also implementing serialization, so we need to either serialize into PEM/DER or hex bytes. Since we're the only ones that are going to be reading this, I have no preference. Do you?

Reply:
I think hex is fine here, I'm not sure there is an ASN.1 structure for a detached EdDSA signature...